packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -237,24 +237,21 @@ export const announceTheme = async ({
  * announce runtime translations by injecting them into the getTextPlugin
  *
  * @param vue
- * @param supportedLanguages
- * @param translations
+ * @param options
  */
 export const announceTranslations = ({
   vue,
-  supportedLanguages,
-  translations
+  ...options
 }: {
-  vue: VueConstructor
-  supportedLanguages: unknown
-  translations: unknown
-}): void => {
-  vue.use(getTextPlugin as any, {
-    availableLanguages: supportedLanguages,
-    defaultLanguage: navigator.language.substring(0, 2),
-    translations,
-    silent: true
-  })
+  vue: App
+} & Partial<GetTextOptions>): void => {
+  vue.use(
+    createGettext({
+      defaultLanguage: navigator.language.substring(0, 2),
+      silent: true,
+      ...options
```

Comment:
can you put the options spreading upfront? My personal taste :)

Reply:
In this case we want options to possibly overwrite the defaults, so we need it in this order. The other cases above are valid though, I adjusted it 👍